manual_user_install.sh

```diff
@@ -0,0 +1,265 @@
+
+    # install required skills
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-volume
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-unknown
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-stop
+    pip3 install git+https://github.com/OpenVoiceOS/skill-alerts
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-personal
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-naptime
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-date-time
+
+    # You can uncomment these lines if the deprecation notices are flooding your logs or slowing the system
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/signal.py
+
+    if [[ ! -d $HOME/.config/mycroft ]]; then
+        mkdir -p $HOME/.config/mycroft
+    fi
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/mycroft.conf $HOME/.config/mycroft/
+    echo $SCRIPT_DIR
+    echo
+    echo "Done installing OVOS core"
+    }
+
+function install_systemd (){
+    echo
+    echo "Installing systemd files"
+    echo
+
+    # install the hook files
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos-systemd-skills $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-core/02-messagebus/files/ovos-systemd-messagebus $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-audio/01-speech/files/ovos-systemd-audio $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-audio/02-voice/files/ovos-systemd-dinkum-listener $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-phal/01-user/files/ovos-systemd-phal $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-phal/02-admin/files/ovos-systemd-admin-phal $HOME/.local/bin
+
+    chmod +x $HOME/.local/bin/ovos-systemd*
+
+    # sdnotify is required
+    pip3 install sdnotify
+
+    # install the service files
+    if [[ ! -d $HOME/.config/systemd/user ]]; then
+        mkdir -p $HOME/.config/systemd/user
+    fi
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos-skills.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-core/02-messagebus/files/ovos-messagebus.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-audio/01-speech/files/ovos-audio.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-audio/02-voice/files/ovos-dinkum-listener.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-phal/01-user/files/ovos-phal.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-phal/02-admin/files/ovos-admin-phal.service $HOME/.config/systemd/user/
+
+    ed $HOME/.config/systemd/user/ovos-admin-phal.service << DONE
```

Comment:
Instead of getting rid of the `User` value, can it be changed to `$USER`.  And the User also needs `sudo` privileges, preferably `no password` at least with that service.  I know it can be done, just haven't looked at it closely

Reply:
Let me look more deeply into why "User" wasn't working. My recollection is that it was trying to start or join a group and didn't have permission. Then it looked to me that we didn't need that group anyway. If we go to system rather than user services this issue may be easier to solve.

Reply:
Is this working?  I was looking into how buildroot handles the admin phal and it is a user service with `User=root` in the `.service` file

Reply:
This is working with no error messages, but now I don't get any audio out.  The log files correctly parse my speech, and the OVOS comes up with the relevant time/date/weather text, but no sound comes out.  The .wav file is created, and I can play it with aplay... And the wave file is correct. Just no audio.  Also intermittent problems getting answers from the tts servers - returning 500 or 502 rather than 200 and a wave file.  I have tried both usb audio and speaker port.

Reply:
An update... I tried the version at https://github.com/OpenVoiceOS/raspbian-ovos.git and now it doesn't run on my 3B. It ran last time I installed it.  (I had to add in a ramdisk).  So I don't think I'm having a configuration issue.  So then I installed raspbian-ovos-dev_1688558201.zip, and I get the same behavior. I can use ovos-speak, but any voice questions are not answered.  I have included the audio and voice logs from 1688558201 if you'd care to have a look.  I am open to suggestions (such as - is it plugged in?)  My intention is to work backwards in the images until I find one that works and then look through the git changes. Yes, it is plugged in. :)

Reply:
I see you are trying YouTube. Can you try a simple "what time is it" with your voice

Reply:
I'm not trying YouTube.  I've learned that the first query always contains what seems to be a random text string that can generate surprising results. Acts like an un-initialized array, but not sure how this happens with python.  Anyway, raspbian-ovos-dev_1688509994.zip works fine for me.  Here are the logs for dev_1688509994.  The pattern is "what time is it" (a throwaway), then "what time is it", and then "what day is it".  I will post parallel results for 1688558201 later this evening.
[audio.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065343/audio.log)
[voice.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065344/voice.log)

Reply:
We have some luck with 1688558201.  The first time I ran it, it worked.  The second time it did not.  
1688558201 when it works:
[audio.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065436/audio.log)
[voice.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065437/voice.log)

1688558201 when it doesn't work:
[audio.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065440/audio.log)
[voice.log](https://github.com/OpenVoiceOS/raspbian-ovos/files/12065441/voice.log)